internal/procinfo/pidtouser_test.go

```diff
@@ -0,0 +1,30 @@
+package procinfo
+
+import (
+	_ "embed"
+	"strings"
+	"testing"
+
+	"github.com/gpuctl/gpuctl/internal/passwd"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// NOTE: could not import same file in a different module, dont want to hardcode
+//
+//go:embed testdata/ashtabula
+var ashtabula string
+
+func loadAshtabula(t *testing.T) passwd.Passwd {
+	t.Helper()
+	users, err := passwd.Parse(strings.NewReader(ashtabula))
+	require.NoError(t, err)
+	return users
+}
+
```

Comment:
Could you add a test that calls `.Get`/`.PidToUser` on [os.Getpid](https://pkg.go.dev/os#Getpid)? (Probably with a map that sets this PID to a preset value, rather than passing /etc/passwd)